docs/framework/01intro/introduction.md

```diff
@@ -1,33 +1,26 @@
 
-**Modalities of Cross-chain Communication**: ...
+## Types of Cross-chain Interaction
+Broadly defined, cross-chain protocols enable the exchange of data and value across chains through one of the following three types of interactions:
 
-- Asset Exchange
+-  **Asset Exchange:** The transfer of ownership of an asset in one network for a corresponding change of ownership in another.
+- **Asset Transfer:** The movement of the value of an asset from a source ledger to a destination ledger. This typically involves locking an asset in the source ledger and minting a synthetic representation on the destination ledger. 
+- **General-purpose Messaging:** The transfer of arbitrary data across chains for the purposes of orchestrating general cross-chain application behaviors. 
 
-- Asset Transfer
-
-- General Purpose Communication:
-
-## Security Challenges
-At its essence, cross-chain communications create dependency relationships between two or more networks. Such dependency relationships typically involve state change in one network driving state change in another. These relationships can be _unidirectional_ or _bidirectional_, _transient_ or _persistent_.  The goal of cross-chain protocols is to enable and guarantee the integrity of these dependencies. To this end, given two networks, a source network and a destination network, where the state in the destination network is dependent on the state in the source network, cross-chain protocols must satisfy the following core properties:
-
-1. only states that are valid in the canonical ledger of the source network are communicated to the destination
+## Security Risks
+At its essence, cross-chain communication creates a dependency relationship between two or more networks. Such dependency relationships typically involve state change in one network driving state change in another. These relationships can be _unidirectional_ or _bidirectional_, _transient_ or _persistent_. The goal of cross-chain protocols is to enable and guarantee the integrity of these dependencies. Specifically, given two networks, a source, and a destination, where the state in the destination network is dependent on the state in the source network, cross-chain protocols must guarantee the following core security properties:
 
+1. only states that are valid and final in the source network are communicated to the destination network
```

Comment:
Each point should start with a capital to match what has been done in other places in the doc

Reply:
Thanks. Fixed.